Compile with -fPIC also on FreeBSD (it's needed for amd64-fbsd)

--- configure.in.orig
+++ configure.in
@@ -959,7 +959,7 @@
 	MAN_MISC_INFO=5
 	;;
 
-linux*)
+linux*|freebsd*)
 	V_CCOPT="$V_CCOPT -fPIC"
 	;;
 
